src/Storages/MergeTree/MergeTreeIndexFullText.cpp

```diff
@@ -606,8 +615,67 @@ bool SplitTokenExtractor::next(const char * data, size_t len, size_t * pos, size
 {
     *token_start = *pos;
     *token_len = 0;
+
     while (*pos < len)
     {
+#if defined(__SSE2__)
+        // NOTE: we assume that `data` string is padded from the right with 15 bytes.
```

Comment:
This is a bug - you cannot make such assumption, see https://github.com/ClickHouse/ClickHouse/issues/19233